### ISSUE TITLE
OBS 28

### ISSUE THREAD

**Issue by bullworkz** [HoneyHazard/PixelMatchSwitcher#34]
Anyone know if this works with OBS 28?  Thanks!

**Comment by phill-85** [comment]
I don't think so, the readme states that it needs a specific fork of obs (https://github.com/HoneyHazard/PixelMatchSwitcher#atomic-effects-fork-of-obs) but that fork hasn't been updated in almost a year....

**Comment by color1519** [comment]
I would really like to get an update to 29 OBS, you did such a job and scored on it, it's sad

**Comment by sbutylkov** [comment]
So sorry y'all! Barely keeping up with my primary project. Unfortunately, not all details of the build process for PM can be disclosed... And we asked to merge some of the features we need into OBS mainline and were rejected at the time; they were working a huge rework of OBS at the time and we got rejected. Unless OBS developers eventually decide to merge atomic counter features the special version of OBS needed for the plugin has to be manually rebuilt, and in order for it to include all streaming plugins the details cannot be publicly disclosed... If somebody could step up to be the main builder/main developer in charge I would share the build details with them. Until then I have to pull myself away from necessary development on another project! I will do my best to take a look at a new build this weekend. Please don't hate me if I don't get to it or run into difficulties!

**Comment by campagne999** [comment]
> So sorry y'all! Barely keeping up with my primary project. Unfortunately, not all details of the build process for PM can be disclosed... And we asked to merge some of the features we need into OBS mainline and were rejected at the time; they were working a huge rework of OBS at the time and we got rejected. Unless OBS developers eventually decide to merge atomic counter features the special version of OBS needed for the plugin has to be manually rebuilt, and in order for it to include all streaming plugins the details cannot be publicly disclosed... If somebody could step up to be the main builder/main developer in charge I would share the build details with them. Until then I have to pull myself away from necessary development on another project to run through the build process (and, occasionally, there are merge conflicts with mainline OBS that need to be resolved)! I will do my best to take a look at a new build this weekend. Please don't hate me if I don't get to it or run into difficulties!

Any update on this? Really would love to have an updated version for 29 for streams. Have some plugins that only work with OBS 29 and this only works on 27. Appreciate your time, and it's all good if you can't

**Comment by sbutylkov** [comment]
Working on it in spare time, and very close to a successful build based on the latest OBS

They have changed a lot in OBS and build dependencies recently. Hoping to have it working in a few days.

**Comment by campagne999** [comment]
That is great to hear! Looking forward to it.
